### PR TITLE
Created Home Hero Section 2 Component

### PR DESCRIPTION
**Pull request by ebacolod**
This pull request aims to resolve issue #124. I have created the HomeHeroSection Component.

## New Feature
- HomeHeroSection2 Component
  - This new component allows users to add a hero section with a secondary style for their website homepage.

## Screenshots of New Feature
- All images portray the new component

### HomeHeroSection2 (Dark)
hhs2_dark

### HomeHeroSection2 (Light)
hhs2_light

### Code Coverage
image

### HomeHeroSection2.jsx
- Added the HomeHeroSection2 component (this is the function that is used to render the component in the PreviewPanel)

### HomeHeroSection2Options.jsx
- Added the HomeHeroSection2Options component (this is the function that is used to render the component in the ComponentOptionsTab)

### HomeHeroSection2.test.jsx
- Added a test to make sure that the words passed in through the options prop are displayed correctly when the component is rendered
- Added a test to make sure that the inputs in the HomeHeroSection2Options component are rendering the correct values
- Added 3 tests to make sure that each onChange function in the HomeHeroSection2Options component work correctly

### photo.jpg
- Image used as a placeholder for the HomeHeroSection2 component

### HomeHeroSection2Dark.png
- Image used to display the preview of the dark mode component in the ComponentsListTab

### HomeHeroSection2Light.png
- Image used to display the preview of the light mode component in the ComponentsListTab

### registeredComponents.js
- Added HomeHeroSection2 component to the array

### CODE REVIEW

**Review by carruthjhall — APPROVED** [review]
The site is responsive for all screen sizes.

All the options are editable and don't break when multiple lines are added. In addition, a photo is able to uploaded and the upload functionality works.

image
Able to get match the coverage you got from my end.

The only thing I would modify is making the button able to work by clicking the button itself, rather than only the text inside, and maybe open in a new tab. However, those are purely QoL suggestions and don't affect the overall component.

File: src/components/HomeHeroSection2/HomeHeroSection2.jsx [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    function Button({index, button}){
+        const {text, link} = button;
+        return (
+            <button className={`max-w-[340px] md:max-w-[821px] lg:max-w-[640px] w-max
+                                rounded-[50px] px-[40px] py-[20px]
+                                break-words font-semibold
+                                font-secondary leading-[22px] text-[18px] 
+                                ${(index == 0 ? `text-white dark:text-black bg-black dark:bg-white` : `text-black dark:text-white bg-transparent border-2`)}`}>
+                <a href={link}> {text} </a>
+            </button>
```
Comment (carruthjhall) [inline src/components/HomeHeroSection2/HomeHeroSection2.jsx:13]: You can wrap the whole button element in the anchor, leaving just the text element inside. That way the whole button can be clicked

**Comment by Lettuce05** [comment]
Had to close and reopen pull request in order to run GitHub action. Action would not run on preexisting pull requests.

**Review by arroya2 — APPROVED** [review]
The component works as intended with know noticeable bugs on my end. Light and Dark mode both work as well. And the component is optimized for different screen sizes.

All five tests passed when I ran the test file for this component

**Pull request merged**